### ISSUE TITLE
Accessibility - Individual Country - Full Report Screen - TalkBack Not Reading PDF Content

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#65]
Because of a technology issue between accessible PDFs and the operating system TalkBback cannot read the PDFs content correctly. By that I mean if you set up headings or data table or have images TalkBack will not read them as such but only as content with no semantic meaning. Screen reader users can use headings to navigate around a page as a sighted user would look for clues visually to find certain things.

I'm going to close the older issue #38, so we don't have all these extra notes to wade through. 

We will keep this issue open to check back now and again to see if Android has corrected the problem.

@carolynhuang,

Can you send an example or two of the individual country PDFs so I can verify that they are accessible? I have not looked at any of them in a year or more and want to make sure they are Section 508 compliant for when the Android and TalkBack problem is corrected.

**Comment by jfc3-dol** [comment]
Because PDFs can't be adequately read meaning the added structure, I saw you added an option to open the PDF in an external application which gives me a list of four options. Is there one or more that you tested the PDF with? If so which ones and I will do the same. If you haven't, I will go back and check them all.

**Comment by jfc3-dol** [comment]
We used our Samsung Galaxy S7 (SM-G930V) to do this testing with.

**Comment by jfc3-dol** [comment]
Need to look to see if the PDFs are accessible using JAWS on my desktop first.

**Comment by jfc3-dol** [comment]
Closing issue because Android and iOS still don't read the structure (headings, TABLE, etc.) from PDFs.
